src/app/psat/modify-conditions/modify-conditions-tabs/modify-conditions-tabs.component.ts

```diff
@@ -202,7 +202,7 @@ export class ModifyConditionsTabsComponent implements OnInit {
     if (this.compareService.modifiedPSAT) {
       let tmpModificationOperationsForm: FormGroup = this.pumpOperationsService.getFormFromObj(this.compareService.modifiedPSAT.inputs, false);
       validModTest = tmpModificationOperationsForm.valid;
-      isDifferent = this.compareService.checkOperationsDifferent();
+      isDifferent = this.compareService.checkOperationsDifferent() || this.compareService.co2SavingsDifferent.getValue().totalEmissionOutputRate;
```

Comment:
This is throwing an error when I first go to Modify All Conditions tab in an assessment with an existing assessment.

Also need to make sure this works properly when switching between assessments

![image](https://user-images.githubusercontent.com/10789740/158859925-c6e1a253-a67a-4794-8d1c-b79bc68d4c6c.png)